cypress/integration/rancid_tomatillos_spec2.js

```diff
@@ -0,0 +1,63 @@
+
+describe('Rancid Tomatillos', () => {
+
+  beforeEach(() => {
+    cy.visit('http://localhost:3000')
```

Comment:
We'll have to think of a way to implement this beforeEach for your tests but not mine when we merge.